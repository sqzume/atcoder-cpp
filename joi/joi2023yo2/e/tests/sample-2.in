5 7
10 13 14 7 12
1 5
2 5
3 3
3 4
2 5
3 1
3 2
