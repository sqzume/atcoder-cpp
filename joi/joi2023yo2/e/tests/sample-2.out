12
7
9
11
